renkuaqs/plugin.py

```diff
@@ -99,7 +99,9 @@ def pre_run(tool):
     # we print
     print(f"\033[31mhere we will prepare hooks for astroquery, tool given is {tool}\033[0m")
 
-    # TODO: where to get renku.client and dir?
+    pwd_env_var = os.environ.get("$PWD", os.getcwd())
+
+    os.environ["PYTHONPATH"] = f"{pwd_env_var}/..:" + os.environ.get('PYTHONPATH', "")
```

Comment:
it's better to write sitecustomize in special directory, maybe subdirectory of .renku. and se pythonpath there

Reply:
`sitecustomize.py` is now written in the subdirectory of `.renku`, `aq`